plugins/main/bridge/bridge.go

```diff
@@ -452,6 +531,10 @@ func cmdAdd(args *skel.CmdArgs) error {
 		return fmt.Errorf("cannot set hairpin mode and promiscuous mode at the same time")
 	}
 
+	if n.vlans != nil && len(n.vlans) > 0 && isLayer3 {
```

Comment:
nit: I think `if n.vlans != nil && len(n.vlans) > 0` is redundant; len can handle a nil slice.

Reply:
I will fix this part.